README.md

```diff
@@ -90,6 +90,31 @@ UI example:
 
 <img src="https://user-images.githubusercontent.com/1560405/110870435-ba91f900-82cc-11eb-9ebc-442152fd67f1.png" width="500"/>
 
+### HomeKit integration
+
+If you want to integrate your alarm with the Apple ecosystem to use Siri or Apple Home automations follow these steps:
+
+1) Add these entries inside configuration.yaml to let Home Assistant create a new bridge with just the Alarm Panel exposed.
+
+```
+homekit:
+  - filter:
+      include_domains:
+        - alarm_control_panel
+    entity_config:
+      alarm_control_panel.alarm_panel:
```

Comment:
We don't need to make any change, but with the next release this name is going to change (we'll have `econnect_metronet` namespace). Don't change anything right now, but let's not forget about it!

Reply:
Domains section should remain the same but let's keep on eye on it.